.travis.yml

```diff
@@ -23,6 +23,8 @@ jobs:
     - stage: check
       env: CMD="+Test/compile"
       name: "Compile all code with Scala 2.12 and 2.13"
+    - env: CMD="scalafmtCheckAll scalafmtSbtCheck"
```

Comment:
@ennru maybe this should be a single job in `check` stage. Although I think that having multiple jobs made it clear what is failing.

Reply:
By the way, just noticed that we are not checking file headers yet (should be another PR though).

Reply:
By the way, I just noticed that we are not checking file headers yet (should be another PR though).